data/wanderer/wanderers start.txt

```diff
@@ -1330,8 +1338,8 @@ mission "Wanderers: Alpha Surveillance F"
 			choice
 				`	"Yes."`
 					goto jump
-				`	"Wait, you know about the Hai?"`
-			`	Danforth looks bemused. "Living on their doorstep? Of course I do. Why do you think the Navy maintains such a large base on Farpoint in the first place? It's not like the human anarchists up north pose that much of a threat."`
+				`	"You seem to be very familiar with Hai space?"`
```

Comment:
I'm not sure saying Danforth is "very familiar" with Hai space necessarily fits here. The only thing he's mentioned is that he knows about the Unfettered, and he himself seems to feel that his knowledge might be outdated.

Reply:
I'm inclined to say that's a problem for the Wanderer's focused PR to deal with (when it happens)

Reply:
Also, he knows the boundaries of Hai space on a map, which does not appear to have political ownership labelled, and can identify "rebel" systems at a glance from recall. That's a fair degree of familiarity.

On reflection; I'm resolving this as firmly a Wanderer's PR problem if people want to debate the subtleties of that then.